app/main.py

```diff
@@ -1,3 +1,52 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: type("Distance") | int) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: type("Distance") | int) -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        self.km += other
+        return self
+
+    def __mul__(self, number: int) -> "Distance":
+        return Distance(self.km * number)
+
+    def __truediv__(self, number: int) -> "Distance":
+        return Distance(round(self.km / number, 2))
```

Comment:
u can use `Distance` without quotes for type annotations

Reply:
It won't let me do this. I get an error saying: "Unresolved reference Distance", and it suggests creating class Distance in main.py.